cmd/feed-ingress/main.go

```diff
@@ -84,13 +85,14 @@ func init() {
 func main() {
 	flag.Parse()
 	cmd.ConfigureLogging(debug)
-
 	lb := createLB()
+	frontend := elb.New("eu-west-1", "test")
```

Comment:
Won't this break if we merge it in to master? Can we add a flag to disable / enable frontends? Either with `--expected-frontends=0`  or by having a blank cluster name or something.